index.js

```diff
@@ -7,10 +7,7 @@ module.exports = function (options) {
     , workers = []
     , restarting = false
 
-  var numWorkers = _.isFunction(options.numWorkers)
-    ? options.numWorkers()
-    : options.numWorkers
-    || 2
+  var numWorkers = _.result(options, numWorkers, 2);
```

Comment:
lodash 2.4.1 doesn't have the defaultValue arg, you'd need to update package.json to use this.

Also, quotes around `numWorkers` - needs to be a string